code/modules/spells/spell_types/devil_boons.dm

```diff
@@ -25,6 +25,7 @@
 					new /obj/item/stack/sheet/mineral/diamond(user.drop_location()),
 					new /obj/item/holochip(user.drop_location(), 1000))
 			C.put_in_hands(item)
+		C.adjustBruteLoss(4)
```

Comment:
I preferred the cell damage penalty you suggested.
This also better matches the Green ruin from Lavaland.